src/BloomBrowserUI/bookEdit/readerSetup/readerSetup.io.js

```diff
@@ -146,7 +146,9 @@ function getChangedSettings() {
         var stage = new ReaderStage((i + 1).toString());
         var row = stages[i];
         stage.letters = row.cells[1].innerHTML;
-        stage.sightWords = row.cells[2].innerHTML;
+
+        // remove punctuation from the list of sight words
+        stage.sightWords = (row.cells[2].innerHTML).replace(/,/g, '');
```

Comment:
(a) not true...you only remove commas. Do you actually need to remove other punctuation?
(b) a more helpful comment would indicate WHY commas need to be removed.